src/main/resources/templates/macros/report/elements.vm

```diff
@@ -39,6 +39,8 @@
                     </tr>
                   #end
                 </table>
+              #elseif ($step.getDocString())
```

Comment:
Good to know that, I was thinking Doc String and Rows are mutually exclusive, maybe because I've never thought to use it in that way :)

For the expandable, I've also thought about. Just like the rows, if it's actually too long, then it definitely makes sense. Otherwise it would also be a pain to manually expand it for every small piece of thing...